<launch>

<param name="fato/model/h5_file" type="str" value="$(find fato_tracker_nodes)/../data/models/nepia/nepia_features.h5"/>
<param name="fato/model/obj_file" type="str" value="$(find fato_tracker_nodes)/../data/models/nepia/nepia.obj"/>
<param name="fato/camera_info_url" value="file://$(find tracker_cameras)/config/logitech_hires.yaml"/>
<rosparam command="load" file="$(find fato_tracker_nodes)/config/tracker_parameters_vx.yaml"/>

<!-- uncomment to use the medium version-->
 <node name="tracker" pkg="fato_tracker_nodes" type="track_node_vx" output="screen">
  <remap from="tracker/image_raw" to="image_raw" />
  <remap from="tracker/camera_info" to="camera_info" />
 </node>
  
</launch>
